avocado/core/result.py

```diff
@@ -107,11 +107,6 @@ def __init__(self, job):
         self.interrupted = 0
         self.tests = []
 
-        # Where this results intends to write to. Convention is that a dash (-)
-        # means stdout, and stdout is a special output that can be exclusively
-        # claimed by a result class.
-        self.output = None
-
```

Comment:
There are also `self.output` references in `avocado.core.remote.result`, please take care of them as well.